authorization_services/topics/resource-create.adoc

```diff
@@ -22,10 +22,10 @@ A string uniquely identifying the type of a set of one or more resources. The ty
 For example, the default type for the default resource that is automatically created is `urn:resource-server-name:resources:default`
 
 [[_resource_create_uri]]
-* *URI*
+* *URIS*
 +
-A URI that provides the location/address for the resource. For HTTP resources, the URI
-is usually the relative path used to serve these resources.
+URIS that provides the locations/addresses for the resource. For HTTP resources, the URIS
+Are usually the relative paths used to serve these resources.
```

Comment:
```Are`` should be in lower case?

Reply:
Nice catch, @pedroigor! I can't believe I missed that. Thank you! Yes, it should be lower case.

Reply:
@matthewhelmke I'm learning :)